src/components/Calculator.js

```diff
@@ -1,57 +1,192 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import '../Style.css';
 
-export default class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
+const Calculator = () => {
+  const [state, setState] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
 
-  handleClick = (event) => {
-    this.setState((state) => calculate(state, event.target.textContent));
-  }
+  const handleClick = (event) => {
+    setState((prevState) => ({
+      ...prevState,
+      ...calculate(prevState, event.target.textContent),
+    }));
+  };
 
-  render() {
-    const { next, total } = this.state;
-    return (
-      <div className="main-container">
-        <div className="result-container">
-          <span className="cal-display">{next || total || 0}</span>
+  return (
+    <div className="main-container">
+      <div className="result-container">
+        <span className="cal-display">{state.next || state.total || 0}</span>
+      </div>
+      <div className="buttons-wrapper">
+        <div className="row-one">
+          <button
+            variant="text"
+            type="button"
+            onClick={handleClick}
+            className="btn1"
+          >
+            AC
+          </button>
+          <button
+            variant="text"
+            type="button"
+            onClick={handleClick}
+            className="btn1"
+          >
+            +/-
+          </button>
+          <button
+            variant="text"
+            type="button"
+            onClick={handleClick}
+            className="btn1"
+          >
+            %
+          </button>
+          <button
+            variant="text"
+            type="button"
+            onClick={handleClick}
+            className="btn-signe"
+          >
+            *
```

Comment:
- [ ] Great job with your project 👍 . Unfortunately, when I try to do a multiplication operation it doesn't work instead the page refreshes.
- [ ] You also have two multiplication buttons. Kindly make sure you have just one and it works 🙏 

[Video](https://www.awesomescreenshot.com/video/10466804?key=e4d1bc75fa102c408b8cac6d48189b70)